pegasus/migrations/105_add_contacts_rollup_table.rb

```diff
@@ -0,0 +1,29 @@
+Sequel.migration do
+  change do
+    create_table(:contacts_rollup, charset: 'utf8') do
+      primary_key :id, unsigned: true, null: false
+      String :email
+      Int :dashboard_user_id
+      String :name
+      String :street_address, size: 1024
+      String :city
+      String :state
+      String :country
+      String :postal_code
```

Comment:
Unfortunately, potentially either. It may be geolocated IP (who knows if that is school, home or elsewhere) or the address they reported as some kind of address on a form. Aside from naming this `ambiguous_city` etc not sure how to make that better at the moment.